test/database/postgres.go

```diff
@@ -0,0 +1,107 @@
+package database
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/golang-migrate/migrate/v4"
+	"github.com/jackc/pgx/v5/pgxpool"
+	"github.com/stretchr/testify/require"
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+type InstanceConfig struct {
+	User     string
+	Password string
+	DBName   string
+	Host     string
+	Port     string
```

Comment:
What do you think about making this an `int` instead of a string, for a little more type-safety?

Reply:
updated